WordPress/gutenberg
# Blocks: Regenerate RSS, Search block fixtures

Related: https://github.com/WordPress/gutenberg/pull/7966/files#r250230253
Related: https://github.com/WordPress/gutenberg/pull/13583#discussion_r259921366

This pull request seeks to fix and regenerate the fixtures for the Search and RSS blocks, which were not correct in their introductions in #13583 and #7966.

Fixtures were regenerated using `npm run fixtures:regenerate`

The original markup for the RSS block fixtures was corrected to refer to `itemsToShow`, `blockLayout` attributes instead of `postsToShow`, `postLayout`. I suspect these changed as part of iterations from the original pull request.

**Testing instructions:**

Verify that the the original markup and serialized output from the fixtures _makes sense_ given what would be expected for the block to store / generate.

Ensure full content test passes:

```
npm run test-unit test/integration/full-content/full-content.spec.js
```
